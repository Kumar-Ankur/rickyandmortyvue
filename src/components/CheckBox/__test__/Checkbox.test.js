import { shallowMount } from "@vue/test-utils";
import Checkbox from '../Checkbox.vue';

describe('check checkbox component', () => {
    test('generate snapshot test', () => {
        const wrapper = shallowMount(Checkbox)
        expect(wrapper.html()).toMatchSnapshot()
    })

    test('check handleclick event', () => {
        const wrapper = shallowMount(Checkbox)
        const inp = wrapper.find('input')
        inp.trigger('click')
        expect(wrapper.isVueInstance()).toBe(true)
    })
})