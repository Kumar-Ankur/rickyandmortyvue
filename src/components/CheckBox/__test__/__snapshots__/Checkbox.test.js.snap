// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`check checkbox component generate snapshot test 1`] = `"<div><input type=\\"checkbox\\" value=\\"\\"> <label></label> <br></div>"`;
